chapters/synchronization.txt

```diff
@@ -4855,7 +4855,7 @@ family.
 The pname:srcQueueFamilyIndex parameter of the barrier must: be set to the
 source queue family index, and the pname:dstQueueFamilyIndex parameter to
 the destination queue family index.
-pname:dstStageMask is ignored for such a barrier, such that no visibility
+pname:dstAccessMask is ignored for such a barrier, such that no visibility
```

Comment:
At some point should be documented in the VUs...

Reply:
Oof, yes. It looks like right now it's implicitly always got to be a valid set of VUs... I can attempt to fix this too whilst I'm here.

Reply:
Done.